src/AppBundle/Security/CodeVoter.php

```diff
@@ -150,8 +150,9 @@ private function canDelete(Code $code, User $user)
     private function isLocationOk()
     {
         $ip = $this->container->get('request_stack')->getCurrentRequest()->getClientIp();
+        $checkIps = $this->container->getParameter('enable_place_local_ip_address_check');
         $ips = $this->container->getParameter('place_local_ip_address');
         $ips = explode(',', $ips);
-        return (isset($ip) and in_array($ip, $ips));
+        return (isset($checkIps) and !$checkIps) or (isset($ip) and in_array($ip, $ips));
```

Comment:
```suggestion
        return $checkIps and !$checkIps) or (isset($ip) and in_array($ip, $ips));
```

en fait, si le param est set à `false`, le isset($param) va renvoyer... `true` !

Reply:
idem pour les autres.
j'ai testé sur des param existant, mais je veux bien que tu teste pour être sûr

Reply:
oula je suis fatigué, je viens de voir qu'il y avait ensuite un `!$param` dans ton code :facepalm:

Reply:
À vrai dire, je ne sais pas si le isset est requis, je l'ai juste mis au cas où comme sur le check existant.
Je suppose que PHP va lancer une exception si `$checkIps` est null dans `!$checkIps`. Ça fait des années que je n'avais pas eu à me préoccuper du concept de `null` :fearful: